addon/utils/member-action.js

```diff
@@ -1,12 +1,12 @@
 import Ember from 'ember';
-import buildOperationUrl from './build-url';
+import { buildOperationUrl } from './build-url';
 
 const { merge } = Ember;
 
 export default function instanceOp(options) {
   return function(payload) {
     let modelName = this.constructor.modelName || this.constructor.typeKey;
-    let requestType = options.type || 'PUT';
+    let requestType = (options.type || 'PUT').toUpperCase();
```

Comment:
This is the fix. ember-data cares about whether the developer uses `'get'` vs `'GET'`